src/components/tabs/themes/light/tab-panel.base.scss

```diff
@@ -2,6 +2,10 @@
 @use '../../../../styles/utilities' as *;
 
 :host {
+    height: 100%;
```

Comment:
Maybe:
`part='content'` should be flex
and tab-panel should be `1 1 auto` like in [igniteui-angular !11515](https://github.com/IgniteUI/igniteui-angular/pull/11515/files).

Reply:
making tab panel `flex: 1 1 auto` will require the content to be `display: flex`, not sure if that is OK.

Reply:
Angular component was implemented with flex end to end, because it has the benefit of being able to work in both scenarios:
 - accommodate to content
 - stretch to parent layout

Depending on what you want. When not using flex, usually you can get only one of them working correctly.

You can test the bug of https://github.com/IgniteUI/igniteui-angular/issues/11479 with this implementation and see if it handles it properly.